docs/preview/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Home"
+layout: default
+permalink: /
```

Comment:
This should be removed

Reply:
No, this is preview, will be moved instead of the index. So has to be present here.

Reply:
If we don't remove this we can't browse to it :) It's part of preview folder so don't think we need this.

Reply:
Removed!